BlazorMarkDownAppJwt/BlazorMarkDownAppJwt/Server/Controllers/MarkDownController.cs

```diff
@@ -0,0 +1,176 @@
+﻿using BlazorMarkDownAppJwt.Server.Services.MarkDowns;
+using BlazorMarkDownAppJwt.Shared;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
+using System.Net;
+
+namespace BlazorMarkDownAppJwt.Server.Controllers
+{
+    [ApiController]
+    public class MarkDownController : Controller
+    {
+        private IDocumentService markDownService { get; }
+
+        public MarkDownController(IDocumentService markDownService)
+        {
+            this.markDownService = markDownService;
+        }
+
+        [HttpGet]
+        [Route("api/markdown")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(MarkDownModel))]
+        public async Task<IActionResult> Get([FromQuery] long currentId)
+        {
+            try
+            {
+
+                var markDownModel = new MarkDownModel
+                {
+                    Body = string.Empty,
+                };
+
+                var document = await markDownService.GetDocument(currentId);
+
+                if (!string.IsNullOrWhiteSpace(document?.MarkDown))
+                {

```

Comment:
It's a good pratice to return "NotFound" result when the entity is not found.
If you "hide" this behavior by creating empty markdown model you will have to write defensive on the UI side and it will be hard to debug and understand why you don't have a markdown value if it's not found

Reply:
Agree with you, my bad to don't make it at the first implementation.
Fixed in : 
https://github.com/clemalex06/BlazorMarkDown/pull/8/files/7b17cdcde7ad6a7c2d66c22a58e512db49fe3da9..614a92a5082bffb7dba45a98b6d1eef6dfdb552e